addon-sidaneThreadmarks.xml

```diff
@@ -153,8 +153,12 @@
         </xen:if>
       </div>
     </div>
+    
+    <noscript>
+      <a href="{xen:link 'threads/threadmarks', $linkData}" rel="nofollow" class="threadmarksTrigger">{xen:phrase threadmarks}</a>
```

Comment:
Wait I'm getting confused as this wants to link to all of the threadmarks, not just the recent threadmarks.